packages/inter-protocol/src/vaultFactory/liquidation.js

```diff
@@ -16,6 +18,16 @@ const trace = makeTracer('LIQ', false);
 /** @typedef {import('@agoric/time/src/types').CancelToken} CancelToken */
 /** @typedef {import('@agoric/time/src/types').RelativeTimeRecord} RelativeTimeRecord */
 
+const makeCancelToken = makeCancelTokenMaker('liq');
+
+/**
+ * This will normally be set. If the schedule goes sideways, we'll unschedule
+ * all events and unset it. When auction params are changed, we'll restart the schedule
```

Comment:
Excellent solution. Let's get some tests of this. Ideally before merging this PR but for the sake of rapid integration we could follow on with that.